fix: updated analysis_test.go

chetanguptaa opened the pull request:
<!-- 
Thanks for creating this pull request 🤗

Please make sure that the pull request is limited to one type (docs, feature, etc.) and keep it as small as possible. You can open multiple prs instead of opening a huge one.
-->

<!-- If this pull request closes an issue, please mention the issue number below -->
Closes # <!-- Issue # here -->
It doesn't fix any already mentioned issues
## 📑 Description
<!-- Add a brief description of the pr -->
This PR updates analysis_test.go and upgrades its readablity
## ✅ Checks
<!-- Make sure your pr passes the CI checks and do check the following fields as needed - -->
- [ ] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [ ] All the tests have passed

## ℹ Additional Information
<!-- Any additional information like breaking changes, dependencies added, screenshots, comparisons between new and old behavior, etc. -->

AlexsJones commented:
Thanks for the contribution, please update from main and correct to a conventional commit title for the PR!

chetanguptaa commented:
> Thanks for the contribution, please update from main and correct to a conventional commit title for the PR!

is it good now I have changed the pull request title and updated my commit
